javascripts/tagmanagerHelper.js

```diff
@@ -227,22 +228,35 @@
             return tagManagerHelper.updateDebugSiteFlag(newUrl, idContainer, 1);;
         }
         var ajaxRequest = new ajaxHelper();
+        var template = $('<h2>' + _pk_translate('TagManager_UpdatingDebugSiteUrlPleaseWait') + '</h2>');
         ajaxRequest.addParams({
             module: 'API',
+            format: 'json',
             method: 'TagManager.changeDebugUrl',
             idContainer: idContainer,
```

Comment:
guess this line can be removed as well, as the parameter isn't needed.
```suggestion
```

Reply:
@sgiehl removed